src/lib/TVPlayerUI/TVPlayerUI.tsx

```diff
@@ -235,6 +235,12 @@ export const TVPlayerUI: React.FC<TVPlayerProps> = (props) => {
                 }
                 key={index}
                 disabled={button.disabled || buttonMap[button.action].disabled}
+                handleArrowPress={(dir) => {
+                  if (dir === "up") {
```

Comment:
I suggest we also add a new hideControlsOnArrowUp prop and check prop is true in this condition. This is because in some use cases there might be buttons above the controls.

Reply:
Yes, you are right; it should be optional for the user.
So I have added the hideControlsOnArrowUp property, as you said.
Could you please check it again?